### ISSUE TITLE
Moving the base branch to `main`

### ISSUE THREAD

**Issue by mxdvl** [guardian/frontend#22947]
As part of The Guardian's commitment to inclusive language, we are changing the base branch from `master` &rarr; `main` from our git repositories. One of the key to this is making sure no open PRs use `master` as their base branch.

## Main tasks & continuous integration

- [x] Update PRs (see below)
- [x] Change base branch in GitHub
- [x] Update TeamCity builds
- [x] Merge changes to `main` #22979 
- [x] One last `grep "master(?!(class|image))"`
- [x] 🤞 

---

### Open issues, oldest first (ship, close or change base branch)

- [x] #20915
- [x] #21187 
- [x] #21456
- [X] #21476
- [x] #21761
- [x] #21811
- [x] #21901
- [x] #21945
- [x] #22008
- [x] #22049
- [x] #22192
- [x] #22196 [snyk]
- [x] #22243
- [X] #22273 (not against `master`)
- [x] #22297
- [x] #22377
- [x] #22394 [snyk]
- [x] #22457 [snyk]
- [x] #22464
- [x] #22492
- [x] #22544 [snyk]
- [x] #22545 [snyk]
- [x] #22611
- [x] #22700 [snyk]
- [x] #22709 [snyk]
- [x] #22713 [snyk]
- [x] #22724
- [x] #22728
- [X] #22738 (not against `master`)
- [x] #22784
- [x] #22791 [snyk]
- [x] #22801
- [x] #22843 [snyk]
- [x] #22858
- [x] #22866
- [x] #22867
#### &uarr; added a comment —— to remind/ping &darr;
- [x] #22883
- [x] #22897 [snyk]
- [x] #22931
- [x] #22942 

cc. @johnduffell

**Comment by johnduffell** [comment]
it's great that you are thinking about the branch rename for this important repo!

Just to clarify, I guess this issue is to close any PRs that are no longer needed or ship them if they are? That is a good thing to do in itself and should be done as a matter of ongoing good practice.

In terms of the timing, this has the side effect of making the process of flipping over the branch name easier which is always good.

**Comment by mxdvl** [comment]
cc @jorgeazevedo & @amyhughes

**Comment by mxdvl** [comment]
All done now!
